shared/login/forms/intro.render.desktop.js

```diff
@@ -3,46 +3,57 @@
 
 import React, {Component} from 'react'
 import {globalColors, globalStyles} from '../../styles/style-guide'
-import {Text, Icon, Button} from '../../common-adapters'
+import {Text, Icon, Button, Box} from '../../common-adapters'
 
 import type {IntroProps} from './intro.render'
 
-export default class Intro extends Component {
-  props: IntroProps;
-
+export default class Intro extends Component<void, IntroProps, void> {
   render () {
     return (
-      <div style={styles.loginForm}>
+      <Box style={{...stylesLoginForm, marginTop: this.props.justRevokedSelf ? 0 : 45}}>
+        {this.props.justRevokedSelf && <Box style={stylesRevoked}>
```

Comment:
This leads to:
<img width="759" alt="screen shot 2016-04-28 at 5 11 41 pm" src="https://cloud.githubusercontent.com/assets/594035/14905311/5072cfd2-0d64-11e6-912e-3a469ab05b91.png">

because 

```
{"" && foo} 
// => ""
```